datasets/mur.yaml

```diff
@@ -22,14 +22,10 @@ Resources:
     Type: S3 Bucket
 DataAtWork:
   Tutorials:
-    - Title: Python Subsetting Script
-      URL: https://github.com/nasa/podaac_tools_and_services/tree/master/subset_opendap
-      AuthorName: Yibo Ziang
-      AuthorURL: https://podaac.jpl.nasa.gov
-    - Title: Python Reader Software
-      URL:  https://podaac-tools.jpl.nasa.gov/drive/files/allData/ghrsst/sw/generic_nc_readers/python/
-      AuthorName: PO.DAAC
-      AuthorURL: https://podaac.jpl.nasa.gov
+    - Title: Tutorials
```

Comment:
Can we make the title something more descriptive? Jupyter and Binder Notebook Tutorials?

Reply:
yes, let me do that now.